pages/osx/shortcuts.md

```diff
@@ -0,0 +1,20 @@
+# shortcuts
+
+> Use the Terminal app to run shortcuts from command line instead from the Shortcuts app.
+> More information: <https://support.apple.com/guide/shortcuts-mac/run-shortcuts-from-the-command-line-apd455c82f02/mac>.
+
+- Run a shortcut called "Count holidays":
+
+`shortcuts run "Count holidays"`
```

Comment:
```suggestion
`shortcuts run "{{Count holidays}}"`
```

It's better to use placeholder because any valid value can be instead of just `Count holidays`.